### DISCUSSION TITLE
Nuclei does not show any result

### DISCUSSION THREAD

**Discussion by pennywise-01** [projectdiscovery/nuclei#3210] [Q&A] [answered]
### Current Behavior:

My nuclei is not shown any result after adding the -t flag to use the templates

here is my command:
nuclei -c 200 -silent -t templates/nuclei-templates/cves/* -u <targeturl> -o output2.txt

Expected response of course the result will be in the output2.txt file, but there is no result at all its just empty. 

When i try the command like this it shows the output:
nuclei -silent -u <target url> -o output.txt

After added the -t flag suddenly it does not giving me any result

Here is the pic
![image](https://user-images.githubusercontent.com/54774679/212085414-8b4e8c91-b87f-4e1a-b7b1-ac0d87b45d5f.png)

Can anyone help me with this?

I already make sure the path to the templates is correct.

**Comment by ehsandeep** [comment, marked as answer]
@Hafzan-250601 when you run default scan, it runs other informative templates that are getting matched, when you run specific templates, in this case, `cves/` it will display results only when it found matches, if you don't see results that mean that means targets are not vulnerable.

**Reply by pennywise-01** [reply]
Yeah, I discovered that by remove the --silent flag. Sorry for the silly question
